File: code/basic/variables/variables.py
# Global variable declaration.
globalVariable = "Global variable"


class Variable:
    # Static variable declaration.
    staticVariable = "Static variable"

    def __init__(self):
        # Instance variable declaration.
        self.instanceVariable = "Instance variable"

    def showLocalVariable(self):
        # Local variable declaration.
        localVariable = "Local variable"
        return localVariable


v = Variable()
print(globalVariable)
print(v.instanceVariable)
print(v.showLocalVariable())
print(v.staticVariable)

File: code/basic/variables/variables.js
//Global variable declaration
var globalVariable = "Global variable"
function variableFn() {
  //Instance variable declaration
  this.instanceVariable = "Instance variable";
  this.showLocalVariable = function () {
    //Local variable declaration
    var localVariable = "Local Variable";
    return localVariable;
  }
}

//Static variable declaration
variableFn.prototype.staticVariable = "Static variable";

var vb = new variableFn();
console.log(globalVariable);
console.log(vb.instanceVariable);
console.log(vb.showLocalVariable());
console.log(vb.staticVariable);